api/proto/info/info.proto

```diff
@@ -241,6 +241,16 @@ message ZInfoAppInstance {
   string uuid = 1;
   string name = 2;
   string domainName = 3;
+  string taskid = 4;
+}
+
+// ZInfoDeviceTasks - send a summary of tasks so that controller knows
+// how many app instance tasks are still present on the device. Controller
+// can then use it to wait for an app to be fully deleted etc.
+message ZInfoDeviceTasks {
```

Comment:
Hey @zed-rishabh -- this definitely requires more documentation in reflection in our API doc before this could be merged. For example, what is this ID, what is the name. Also, I really don't think status should be a string -- it probably should map to the other probof sturct.

Reply:
I added it in api v2 docs.

Reply:
Cool -- so this is representation of containerd tasks -- thanks for making this clear in the docs, but I have one more question now and I'm sure @deitch will have an opinion or two ;-)